### PR TITLE
Update 2018-10-08-new-in-symplify-5-create-merge-and-split-monorepo-w…

### PR DESCRIPTION
**Pull request by Gappa**
…ith-1-command.md

### CODE REVIEW

File: source/_posts/2018/2018-10-08-new-in-symplify-5-create-merge-and-split-monorepo-with-1-command.md [hunk 1]
```diff
@@ -115,4 +115,4 @@ and read [`README`](https://github.com/symplify/monorepobuilder) for detail usag
 
 <br>
 
-Happy monorapping!
+Happy monorepping!
```
Comment (TomasVotruba) [inline source/_posts/2018/2018-10-08-new-in-symplify-5-create-merge-and-split-monorepo-with-1-command.md]: This one is intended ;)

**Comment by TomasVotruba** [comment]
Thank you :+1:

File: source/_posts/2018/2018-10-08-new-in-symplify-5-create-merge-and-split-monorepo-with-1-command.md (hunk 1 above)
Comment (Gappa) [reply source/_posts/2018/2018-10-08-new-in-symplify-5-create-merge-and-split-monorepo-with-1-command.md]: Ah, I wasn't quite sure about this one :)

Comment (TomasVotruba) [reply source/_posts/2018/2018-10-08-new-in-symplify-5-create-merge-and-split-monorepo-with-1-command.md]: I like to play with words... There is also 3rd typo possible in this word ;)

**Pull request merged**